### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by f3r**
Hi @dshcheung 

Can you please merge this awesome PR

### CODE REVIEW

**Comment by dshcheung** [comment]
Irrelevant to my project. Please remove the stuff i commented on

File: README.rdoc [hunk 1]
```diff
@@ -1,28 +1,10 @@
 == README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## TO BE DONE
```
Comment (dshcheung) [inline README.rdoc]: Irrelevant to my project

File: README.rdoc [hunk 2]
```diff
@@ -1,28 +1,10 @@
...
 
-Things you may want to cover:
+I strongly apologize for not keeping this document up to date
 
-* Ruby version
+We'll be with you shortly.
```
Comment (dshcheung) [inline README.rdoc:5]: here

File: README.rdoc (hunk 1 above)
Comment (f3r) [inline README.rdoc]: I will remove TO BE DONE ASAP

**Comment by paulld** [comment]
:+1:

**Pull request merged**